pkg/cmd/clients/factory.go

```diff
@@ -491,7 +493,18 @@ func (f *factory) CreateVaultClient(name string, namespace string) (vault.Client
 	if err != nil {
 		return nil, errors.Wrap(err, "creating vault client")
 	}
-	vaultClient, err := clientFactory.NewVaultClient(name, namespace)
+	// if there's an issue loading a requirements yaml lets just default to automatic
+	useIngressURL := false
+	requirements, _, _ := config.LoadRequirementsConfig("")
+
+	// allows us to override using the default lookup URL for vault and ensure we always use the ingress. Used in CI.
+	if requirements.Vault.DisableURLDiscovery {
```

Comment:
Just checking 'cos I'm not clear enough on how all this works - will this mean that `jx install --ng` (or `jx create cluster ... --ng` or `--vault --gitops` etc) gets `useIngressURL` as `true`?

Reply:
Just had a chat with @abayer on slack, we're thinking we should remove the `--ng` and `--gitops` flags + disable tests, trying to support those flags is a pita as they never quite worked anyways, `jx boot` should be the way folks install with vault etc

Andrew is going to start raising PRs to avoid the version stream tests for these contexts and we can do more PRs tomorrow to deprecate the flags.